tests/e2e/features/artifacts/transferPage/atrifacts-transfers.feature

```diff
@@ -202,4 +202,30 @@ Feature: Artifacts - UI
     Then Modal card element with the "//*[contains(@class, '-my-0.5')]//button[contains(., 'ETH')]" xpath should be "visible"
     Then Element with "text" "Send to zkSync Era Testnet" should be "visible"
     Then Element with "text" "Send to zkSync Era Testnet" should be "clickable"
-    
\ No newline at end of file
+
+  @id1286 @Transfer
+  Scenario: Check max button functionality
+    When I go to page "/transaction/zksync/era/send/?address=0x2CF4F78917A16C9584AeB5d4c5bD2713d724C75d"
+    Then I confirm the network switching
+      #Max button is displayed
+    Then Element with "partial text" "Max" should be "visible"
+    Then Element with "partial text" "Max" should be "clickable"
```

Comment:
I would suggest to add here and in Allure Hover verification I hover the "text" element with " Max " value

Additionally based on id1539 you can use our method from test solution to grab this value and compare what we have in the input field

Reply:
@pcheremu hover functionality can't be applied here because in this perticular case value on the hover is stored as `title` attribute. And in fact I can implement this check in a next way: 
![Uploading image.png…]()

Reply:
make sense, go ahead